### DISCUSSION TITLE
Welcome to AzuraCast Discussions!

### DISCUSSION THREAD

**Discussion by BusterNeece** [AzuraCast/AzuraCast#3523] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by adamburnsuk** [comment]
Hey, my name is Adam Burns and I'm cofounder of an online podcast live streaming network for LGBT and friendly podcasts.  We host live events and have shows that stream live during the week as well.  Currently we use Libretime, but I'm testing out AzuraCast to switch.  I'm also a developer/product owner by day and will possibly be looking to help out where I can as well.

**Reply by BusterNeece** [reply]
@adamburnsuk Welcome! I wish you the best in your project's efforts, and I hope you find AzuraCast to your liking as well. Let us know if you have any questions or would like to contribute; we always have room for new contributors.
